src/components/molecules/Modal/Modal.tsx

```diff
@@ -55,22 +43,33 @@ export const Modal: React.FC<ModalProps> = ({
   const hasHeader = isTruthy(title || closeButton);
 
   return (
-    <div className={overlayClasses}>
-      <div className={containerClasses} ref={containerRef}>
-        {hasHeader && (
-          <div className="Modal__header">
-            <span className="Modal__title">{title}</span>
-            {closeButton && (
-              <img
-                className="Modal__close"
-                src={PortalCloseIcon}
-                alt="close portal"
-                onClick={onHide}
-              />
-            )}
-          </div>
-        )}
-        <div className="Modal__body">{children}</div>
+    <div
```

Comment:
This one should be moved to `./src/components/admin` so it is not confused with attendee side component

Reply:
This one is also being used by both admin and attendee, that's why I suggested a common folder for components like this one.